src/aura/STG_CMP_Tabs/STG_CMP_Tabs.cmp

```diff
@@ -52,7 +60,7 @@
 
         <div aura:id="afflTabContent" class="site-code--content slds-scrollable--x slds-tabs__content" role="tabpanel">
 
-            <c:STG_CMP_Affl hierarchySettings="{!v.hierarchySettings}" isView="{!v.isView}" namespacePrefix="{!v.namespacePrefix}" />
+            <c:STG_CMP_Affl hierarchySettings="{!v.hierarchySettings}"  affiliationRoleMapValue="{!v.affiliationRoleMapValue}"  affiliationRoleMapLabel="{!v.affiliationRoleMapLabel}"  affiliationRolePicklistEntries="{!v.affiliationRolePicklistEntries}"  affiliationStatusMapValue="{!v.affiliationStatusMapValue}"  affiliationStatusMapLabel="{!v.affiliationStatusMapLabel}"  affiliationStatusDeleteMapValue="{!v.affiliationStatusDeleteMapValue}"  affiliationStatusDeleteMapLabel="{!v.affiliationStatusDeleteMapLabel}"  affiliationStatusPicklistEntries="{!v.affiliationStatusPicklistEntries}" isView="{!v.isView}" namespacePrefix="{!v.namespacePrefix}" />
```

Comment:
consider adding some carriage returns here after the attributes so future devs can see what is being sent in at a glance.